Haneke/Cache.swift

```diff
@@ -177,7 +177,7 @@ open class Cache<T: DataConvertible> where T.Result == T, T : DataRepresentable
     
     // MARK: Formats
 
-    public var formats : [String : (Format<T>, NSCache<AnyObject, AnyObject>, DiskCache)] = [:]
+    open var formats : [String : (Format<T>, NSCache<AnyObject, AnyObject>, DiskCache)] = [:]
```

Comment:
Why does `formats` need to be `open` instead of `public`?

Reply:
Jeff had made that change. not sure at the moment. its in the Haneke version we were using in the app.